lib/sles4sap/sap_deployment_automation_framework/naming_conventions.pm

```diff
@@ -43,6 +43,7 @@ our @EXPORT = qw(
   generate_deployer_name
   get_workload_vnet_code
   get_sdaf_inventory_path
+  get_sut_sshkey_path
```

Comment:
SSH key filename was hardcoded in few other places so I added a common function.